server/src/test/java/org/elasticsearch/index/mapper/DynamicMappingTests.java

```diff
@@ -327,14 +337,17 @@ public void testObject() throws Exception {
         DocumentMapper mapper = parser.parse("type", new CompressedXContent(mapping));
         assertEquals(mapping, serialize(mapper));
 
-        Mapper update = parse(mapper, parser, XContentFactory.jsonBuilder().startObject().startObject("foo").startObject("bar").field("baz", "foo").endObject().endObject().endObject());
+        Mapper update = parse(mapper, parser, XContentFactory.jsonBuilder().startObject().startObject("foo").startObject("bar")
+            .field("baz", "foo").endObject().endObject().endObject());
         assertNotNull(update);
         // original mapping not modified
         assertEquals(mapping, serialize(mapper));
         // but we have an update
         assertEquals(Strings.toString(XContentFactory.jsonBuilder().startObject().startObject("type").startObject("properties")
-                .startObject("foo").startObject("properties").startObject("bar").startObject("properties").startObject("baz").field("type", "text")
-                .startObject("fields").startObject("keyword").field("type", "keyword").field("ignore_above", 256).endObject()
+                .startObject("foo").startObject("properties").startObject("bar").startObject("properties").startObject("baz")
```

Comment:
I tend to try an indent these manually so they *look* a little more like json. Not that this is required, but it does help when they get big like this.

Reply:
But this one is actually pretty huge. I don't think it'd fit!